packages/frontend/src/utils/useDebounce.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from 'react'
+
+export default function useDebounce(value: any, delay: number) {
```

Comment:
This hook looks cool. I think it'll only work when the value is primitive. Might be better if we use explicit typing string | number | boolean.